buidl/mnemonic.py

```diff
@@ -98,6 +99,40 @@ def bytes_to_mnemonic(b, num_bits):
     return " ".join(mnemonic)
 
 
+def dice_rolls_to_mnemonic(dice_rolls, num_words=24, allow_low_entropy=False):
+    """
+    returns a mnemonic from a string of 6-sided dice rolls
+    (>=100 rolls for 24 words recommended)
+    (>=50 rolls for 12 words recommended)
+    """
+    # check that the number of words provided is 12, 15, 18, 21, or 24
+    if num_words not in (12, 15, 18, 21, 24):
+        raise InvalidBIP39Length(
+            f"{num_words} words requested (must be 12, 15, 18, 21, or 24 words)"
+        )
+    # check that valid dice rolls have been provided
+    if not isinstance(dice_rolls, str):
+        raise ValueError("Dice rolls must be provided as a string")
+    if (
+        len([roll for roll in dice_rolls if roll not in ("1", "2", "3", "4", "5", "6")])
```

Comment:
Nit: this is fine, but can be made simpler (and with a clearer error message) without the list comprehension.

```
valid_dice_rolls = set(["1", "2", "3", "4", "5", "6"])
for roll in dice_rolls:
    if roll not in valid_dice_rolls:
        raise ValueError(f"Invalid dice roll: {roll}")
```